src/main/java/com/exasol/errorcodecrawlermavenplugin/crawler/MessageStepReader.java

```diff
@@ -10,22 +11,27 @@
 import spoon.reflect.code.CtInvocation;
 
 /**
- * Reader for invocations of {@link ErrorMessageBuilder#message(String)}.
+ * Reader for invocations of {@link ErrorMessageBuilder#message(String, Object...)}.
  */
 class MessageStepReader implements MessageBuilderStepReader {
-    private static final String SIGNATURE = "message(java.lang.String)";
+    private static final Set<String> SUPPORTED_SIGNATURES = Set.of("message(java.lang.String,java.lang.Object[])",
+            "message(java.lang.String)");
 
     @Override
     public void read(final CtInvocation<?> builderCall, final ErrorMessageDeclaration.Builder errorCodeBuilder,
             final Path projectDirectory) throws InvalidSyntaxException {
         final List<CtExpression<?>> arguments = builderCall.getArguments();
-        assert arguments.size() == 1;
+        assert !arguments.isEmpty();
         final CtExpression<?> messageArgument = arguments.get(0);
-        errorCodeBuilder.prependMessage(new ArgumentReader(SIGNATURE).readStringArgumentValue(messageArgument));
+        final String message = new ArgumentReader(builderCall.getExecutable().getSignature())
```

Comment:
can we extract this to a `getMessageFrom(builderCall)` method or similar?

Reply:
why? it would only be one line in a method?